src/tribol/interface/mfem_tribol.cpp

```diff
@@ -69,6 +69,8 @@ void registerMfemCouplingScheme( IndexT cs_id, int mesh_id_1, int mesh_id_2, con
     }
   }
   coupling_scheme.setMfemMeshData( std::move( mfem_data ) );
+  // scale binning proximity if a higher order mesh is used
```

Comment:
Is this comment correct? This looks like it sets it, but doesn't scale it.

Reply:
This stuff could be made cleaner, but the problem is the binning proximity is set in Tribol but the LOR factor is set in the MFEM interface (which sits above Tribol).  Maybe LOR factor could be moved down to the Tribol level (just to set binning proximity cleaner)?